app/templates/content/accessibility-statement.html

```diff
@@ -0,0 +1,95 @@
+
+      <h2 class="govuk-heading-m">Non accessible content</h2>
+      <p class="govuk-body">The content listed below is non-accessible for the following reasons:</p>
+      
+      <h3 class="govuk-heading-s">Non compliance with the accessibility regulations</h3>
+      <p class="govuk-body">Some pages don’t clearly identify content within the page. This fails <abbr title="Web Content Accessibility Guidelines">WCAG</abbr> 2.1 A success criterion 2.4.2 (title of a Web page not identifying the contents).</p>
+      <p class="govuk-body">Not all our interactive elements meet the required non-text contrast ratio. This fails WCAG 2.1 AA success criterion 1.4.11  (non-text contrast).</p>
+      <p class="govuk-body">Labeled elements on some pages are not declared by screen readers. This fails WCAG 2.1 A success criterion 2.5.3 (label in name).</p>
+      <p class="govuk-body">Not all our journeys offer consistent navigation. This fails WCAG 2.1 AA success criterion 3.2.3 (consistent navigation).</p>
+      <p class="govuk-body">Some error messages include irrelevant text and do not direct screen reader users to an error message link. This fails WCAG 2.1 A success criterion 3.3.1 (error identification).</p>
+      <p class="govuk-body">Search and sitemap functionality is missing. This fails WCAG 2.1 AA success criterion 2.4.5 (multiple ways).</p>
+      <p class="govuk-body">Screen readers do not declare the state or function of some elements correctly. This fails WCAG 2.1 A  success criterion 4.1.2 (name,role,value).</p>
+
+      <h3 class="govuk-heading-s">Disproportionate burden</h3>
+      <p class="govuk-body">We will not seek to offer accessible PDFs due to the technical complexity of making the changes and anticipated migration of the platform.</p>
+
+      <h2 class="govuk-heading-m">Content not within the scope of the accessibility regulations</h2>
+      <h3 class="govuk-heading-s">PDFs and other documents</h3>
+      <p class="govuk-body">All supplier submitted PDFs do not need to comply with the accessibility requirement because they are <a href="http://www.legislation.gov.uk/uksi/2018/952/regulation/4/made" rel="external">third party content</a> that is neither funded nor developed by the Crown Commercial Service. If anyone requires an accessible format of these documents,  contact: <a href="mailto:info@crowncommercial.gov.uk">info@crowncommercial.gov.uk</a></p>
+
+      <h2 class="govuk-heading-m">How we tested this website</h2>
+      <p class="govuk-body">This website was tested for compliance with the Web Content Accessibility Guidelines V2.1 level A and level AA. Tests have been carried out externally and internally.</p>
+
+      <h2 class="govuk-heading-m">What we’re doing to improve accessibility</h2>
+      <p class="govuk-body">We plan to fix all known issues by September 2021.</p>
+      <p class="govuk-body">This statement was prepared on 23 September 2020. It was last updated on 23 September 2020.</p>
```

Comment:
Should be 22nd September if we release this today!

Reply:
Thanks!